pkg/kubernetes/views/group-page.html

```diff
@@ -11,7 +11,7 @@ <h3 ng-if="!group()">
         <i class="pficon pficon-users"></i>
         <span>{{ groupName }}</span>
     </h3>
-    <a ng-href="#/projects" translate>Show all Projects</a>
+    <a tabindex="0" ng-href="#/projects" translate>Show all Projects</a>
```

Comment:
Why is this an "ng-href" instead of just an "href"?

And we shouldn't just invent attributes — that makes the HTML invalid. HTML5 has [data-*](https://w3c.github.io/html/single-page.html#embedding-custom-non-visible-data-with-the-data-attributes) for custom properties.

Reply:
I guess Angular changes this to href and then tabindex probably isn't needed?

Reply:
[According to the docs](https://docs.angularjs.org/api/ng/directive/ngHref), `ng-href` is used in place of `href` for Angular to swap out variables in URIs.

As the URI doesn't include a variable in `{{}}`, it can just be an `href`. I don't want to touch working code that I'm not familiar with, however, so having a `tabindex="0"` would still be useful to cover the case if Angular hasn't had a chance to swap it out for an `href` and someone happened to tab to the link.